src/uiprotect/api.py

```diff
@@ -1175,6 +1209,16 @@ async def get_bootstrap(self) -> Bootstrap:
         data = await self.api_request_obj("bootstrap")
         return Bootstrap.from_unifi_dict(**data, api=self)
 
+    async def get_keyrings(self) -> list[Keyring]:
+        """Gets keyrings list from UFP instance"""
+        data = await self.api_request_list("keyrings")
+        return data
+
+    async def get_ulpusers(self) -> list[UlpUser]:
+        """Gets ulpusers list from UFP instance"""
+        data = await self.api_request_list("ulp-users")
```

Comment:
```suggestion
        return await self.api_request_list("ulp-users")
```

Which version was `ulpusers` added on? If we still support it we may need to handle it being missing